example_2/README.rst

```diff
@@ -0,0 +1,97 @@
+*********
+DiffBot
+*********
+
+*DiffBot*, or ''Differential Mobile Robot'', is a simple mobile base with differential drive.
+The robot is basically a box moving according to differential drive kinematics.
+The *DiffBot* URDF files can be found in ``description/urdf`` folder.
+
+1. To check that *DiffBot* description is working properly use following launch commands::
+
+    ros2 launch ros2_control_demo_example_2 view_robot.launch.py
```

Comment:
I think that you have to add `.. code::` syntax to show this properly in rst.

Reply:
The best would be to test this by pulling `control.ros.org` and then by referencing this files too (as done for other repositories) and checking if generated docs are correct.

(not necessary for the first version – we can fix this later, but you might be curious)

Reply:
I think I have to use `.. code-block:: shell` or `.. code-block:: bash` for proper rendering in the docs (with the copy-content button etc) instead of `::`.
The inline `.. code::` seems not to work